aiosqlite/cursor.py

```diff
@@ -12,18 +12,30 @@ class Cursor:
     def __init__(self, conn: "Connection", cursor: sqlite3.Cursor) -> None:
         self._conn = conn
         self._cursor = cursor
+        self.__chunk: Optional[Iterator[sqlite3.Row]]
```

Comment:
Please don't use double underscore prefixes. This causes name mangling that both makes it difficult to debug from tracebacks and also makes it more difficult when subclassing.